src/CameraHandler.cpp

```diff
@@ -25,8 +25,10 @@ CameraHandler::CameraHandler(std::string uniqueId) :
     try {
         init(uniqueId);
     } catch (...) {
-        release();
-        throw;
+        // if init failed we just make sure that mCamera is empty,
+        // but keep working as camerabe is mandatory service for DomA
+        DLOG(mLog, DEBUG) << "Camera init failed, so we will run in camera-less mode.";
```

Comment:
I would love to see some more meaningful judgement why it is good to run in "cameraless mode"

Reply:
Comment reworked.